app.rb

```diff
@@ -0,0 +1,125 @@
+require_relative 'person'
+require_relative 'base_decorator'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'classroom'
+
+class App
+  attr_accessor :people, :rentals, :books
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  # Create book list
+  def book_list
+    if @books.empty?
+      puts 'There are no books in the list'
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index} - Title: #{book.title.capitalize}, Author: #{book.author.capitalize}"
+      end
+    end
+  end
+
+  # Create people list
+  def people_list
+    if @people.empty?
+      puts 'There are no people in the list'
+    else
+      @people.each_with_index do |person, index|
+        if person.instance_of?(Teacher)
+          puts "#{index} -[Teacher] Name. #{person.name}, ID: #{person.id}, Age: #{person.age}"
+        elsif person.instance_of?(Student)
+          puts "#{index} -[Student] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+        end
+      end
+    end
+  end
```

Comment:
- [ ] It would be nice if you could modify the data displayed for a teacher's age to be their age and not their name as shown in the screenshot below:

<img width="408" alt="Screenshot 2023-04-29 at 12 29 57 PM" src="https://user-images.githubusercontent.com/50941074/235302699-854469a6-e5b3-47f9-9407-a69329a931dc.png">

- [ ] The same thing displays when creating a rental but if you fix the bug in this method, it's all fixed 👍🏽